spec/javascripts/spec_helper.js

```diff
@@ -0,0 +1,11 @@
+//= require support/json2
```

Comment:
It may not be a concern of this PR then, but that's a fairly large change in behavior that will definitely impact a lot of users of Teaspoon / the asset pipeline in general. Ok, I'll create an issue to track that since we'll need to adjust the install generator to support that. If you want to tackle that it would be awesome too, but don't feel obligated since it's only tangentially related.